app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from os import path, makedirs, remove
+from shutil import move
+
+
+def make_directories(destination_path: str) -> None:
+
+    directories = destination_path.split("/")
+    current_path = ""
+
+    for directory in directories:
+        current_path = path.join(current_path, directory)
+        if not path.exists(current_path):
+            os.makedirs(current_path)
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    mv, source_path, destination_path = parts
+    if len(parts) != 3 or mv != "mv":
+        return
+
+    if not path.exists(source_path):
+        return
+
+    is_directory = False
+    if destination_path.endswith("/"):
+        is_directory = True
+        destination_path = destination_path.rstrip("/")
+
+    if is_directory:
+        make_directories(destination_path)
```

Comment:
You could use built-in methods from `os` module such as `isdir` or  `makedirs`. Do not rely on specific path delimiter in code, make it platform-agnostic.